events/README_DynamoDB.md

```diff
@@ -3,27 +3,25 @@
 The following is a sample Lambda function that receives DynamoDB event data as input and writes some of the record data to CloudWatch Logs. (Note that by default anything written to Console will be logged as CloudWatch Logs.)
 
 ```go
-
 import (
-    "strings"
+	"context"
+	"fmt"
 
-    "github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/events"
 )
 
-func handleRequest(ctx context.Context, e events.DynamoDbEvent) {
+func handleRequest(ctx context.Context, e events.DynamoDBEvent) {
 
-    for _, record := range e.Records {
-        fmt.Printf("Processing request data for event ID %s, type %s.\n", record.EventId, record.EventName)
+	for _, record := range e.Records {
+		fmt.Printf("Processing request data for event ID %s, type %s.\n", record.EventID, record.EventName)
 
-        // Print new values for attributes of type String
-        for name, value := range record.Dynamodb.NewImage {
-        	if value.DataType() == ddbevents.DataTypeString {
```

Comment:
`DataTypeString` is defined in `events` package.
https://github.com/aws/aws-lambda-go/blob/28a6e9fa1fc02d45d9a8c0a8d22c7d5c3580aa03/events/attributevalue.go#L142